### PR TITLE
Fix compile on nightly.

### PR DESCRIPTION
**Pull request by brndnmtthws**
LocalHistogramTimer::new_coarse() should take a local histogram as an
arg.

### CODE REVIEW

**Comment by breeswish** [comment]
Thanks a lot. Would you also like to modify our Travis CI so that nightly can be included and checked in CI?

**Comment by brndnmtthws** [comment]
> Thanks a lot. Would you also like to modify our Travis CI so that nightly can be included and checked in CI?

Done!

**Comment by brndnmtthws** [comment]
Woops, not so fast, that doesn't work. One moment.

**Comment by brndnmtthws** [comment]
I think the travis build needs a kick.

**Comment by breeswish** [comment]
Hmm.. Travis status is simply missing.. OMG

File: .travis.yml [hunk 1]
```diff
@@ -37,6 +37,10 @@ matrix:
   - rust: nightly
 
 script:
+  - if [ $TRAVIS_RUST_VERSION = 'nightly' ]; then
```
Comment (breeswish) [inline .travis.yml]: Maybe we can use the include feature: https://docs.travis-ci.com/user/build-matrix/#excluding-jobs

e.g.

```yaml
include:
- rust: nightly
- rust: nightly
  env: FEATURES="nightly protobuf push process"
- rust: stable
```

Comment (brndnmtthws) [reply .travis.yml]: Ah, yeah, that looks cleaner, didn't know about that.

**Comment by brndnmtthws** [comment]
Rebased on master, and updated to use the travis matrix feature instead. Seems travis is back from its hiatus as well.

**Pull request merged**